guillotina/api/service.py

```diff
@@ -45,11 +45,40 @@ class Service(View):
     __validator__ = __schema__ = None
     _sentinal = object()
 
+    def _validate_parameters(self):
+        if 'parameters' in self.__config__:
+            data = self.request.url.query
+            for parameter in self.__config__['parameters']:
+                if parameter['in'] == 'query':
+                    if parameter['schema']['type'] == 'integer':
```

Comment:
Could we handle KeyError here?

Reply:
keyError for parameter['schema']['type']?

Reply:
Yes, if they define a parameter without the `schema` defined

I imagine it already default to string so you don't need to define this.